tests/integration/test_server.py

```diff
@@ -70,7 +70,8 @@ def test_server_can_be_restarted_on_same_port():
 
 
 @pytest.mark.skipif(
-    platform.system() == "Windows", reason="Can only run one server on Windows for some reason."
+    platform.system() == "Windows",
```

Comment:
I assume Black made these changes? I wonder why...

Reply:
Black can be wierd... also, there is a newer version out... maybe it changes how it formats things...